Makefile

```diff
@@ -146,11 +146,24 @@ db:
 upgrade:
 	GOOS=$(HOSTOS) GOARCH=$(HOSTARCH) $(HOSTGO) build $(GOHOSTFLAGS) -o ./bin/syz-upgrade github.com/google/syzkaller/tools/syz-upgrade
 
+# `extract` extracts const files from various kernel sources, and may only
+# re-generate parts of files.
 extract: bin/syz-extract
+ifeq ($(TARGETOS),fuchsia)
```

Comment:
Can't we do something like:
```
$(MAKE) generate_fidl TARGETARCH=amd64
$(MAKE) generate_fidl TARGETARCH=arm64
```
without duplicating its contents? generate_fidl already contains a check on TARGETOS, so should be no-op for all other OSes.

Reply:
good point; fixed!